src/tests/multihost/alltests/sssd_multidomain.conf

```diff
@@ -10,9 +10,11 @@ debug_level = 0xFFF0
 proxy_lib_name = ldap
 proxy_pam_target = sssdproxyldap
 
-[domain/files]
+[domain/local]
```

Comment:
We already have the proxy provider, can we use the same?

Reply:
There is difference between two domains.

Proxy domain is with proxy_lib_name = ldap
local domain is with proxy_lib_name = files